nss-day-cohort-60/rare-client-tenacious-tators
# Mp stylin

Closes Ticket #


# Description of Proposed Changes

Styled My Posts page and comments view

---


# Steps to Test

1. Fetch the `MP-Stylin` branch and switch to it
2. Navigate to MyPosts make sure it looks stellar
3. Navigate to View Comments make sure they look great, too. Images should appear next to author names.
